easybuild/easyconfigs/c/cryoDRGN/cryoDRGN-0.3.2-fosscuda-2020b.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'PythonBundle'
+
+name = 'cryoDRGN'
+version = '0.3.2'
+
+homepage = 'https://cb.csail.mit.edu/cb/cryodrgn/'
+description = """cryoDRGN: Deep Reconstructing Generative Networks for cryo-EM heterogeneous
+ reconstruction.
+ CryoDRGN is a neural network based algorithm for heterogeneous cryo-EM reconstruction. In
+ particular, the method models a continuous distribution over 3D structures by using a neural
+ network based representation for the volume.
+"""
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('IPython', '7.18.1'),
+    ('JupyterLab', '2.2.8'),
+    ('SciPy-bundle', '2020.11'),
+    ('PyTorch', '1.7.1'),
+    ('numba', '0.52.0'),
+    ('scikit-learn', '0.23.2'),
+    ('matplotlib', '3.3.3'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('colorlover', '0.3.0', {
+        'checksums': ['b8fb7246ab46e1f5e6715649453c1762e245a515de5ff2d2b4aab7a6e67fa4e2'],
+    }),
+    ('retrying', '1.3.3', {
+        'checksums': ['08c039560a6da2fe4f2c426d0766e284d3b736e355f8dd24b37367b0bb41973b'],
+    }),
+    ('plotly', '4.14.3', {
+        'checksums': ['7d8aaeed392e82fb8e0e48899f2d3d957b12327f9d38cdd5802bc574a8a39d91'],
+    }),
+    ('cufflinks', '0.17.3', {
+        'checksums': ['48c1b3406dc030004121966489eebc5518cea70fd4e3f16379b491328501a644'],
+    }),
+    ('pynndescent', '0.5.2', {
+        'checksums': ['d9fd22210b8d64368376ff392e876fb72fe3cda282396cfa6a59440ab6600771'],
+    }),
+    ('umap-learn', '0.5.1', {
+        'checksums': ['3e3e5e526109866012a9da79f423c922edc379c6cac9bf65ea08fbb9dd93ff3a'],
+        'modulename': 'umap',
+    }),
+    ('seaborn', '0.11.1', {
+        'checksums': ['44e78eaed937c5a87fc7a892c329a7cc091060b67ebd1d0d306b446a74ba01ad'],
+    }),
```

Comment:
Replace this with a dependency on https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/s/Seaborn/Seaborn-0.11.1-fosscuda-2020b.eb